lib/checkPullRequestBranch.js

```diff
@@ -27,13 +27,44 @@ const checkBranch = async (context) => {
     return branchName.startsWith(prefix);
   });
 
-  if (branchIsInvalid) {
+  if(branchName === 'develop')
+  {
```

Comment:
The formatting here is not correct.
```suggestion
  if (branchName === 'develop') {
```